File: StockManagement/Entities/ConcernPerson.cs
using System;
using System.Collections.Generic;

namespace StockManagement.Entities
{
    public partial class ConcernPerson
    {
        public int ConcernPersonId { get; set; }
        public string ConcernPersonName { get; set; } = null!;
        public int IsDeleted { get; set; }
    }
}

File: StockManagementAngular/MRMS-Client/RSSMApp/src/app/models/concernPerson/concern-person.ts
export class ConcernPerson {
    constructor(
        public concernPersonId?: number,
        public concernPersonName?: string,
        public isDeleted?: number,
      ) { }
}

export interface ConcernPersonMapping{
  id: number,
  concernPersonId: number,
  companyId: number,
  companyName: string
}
